src/components/Search/utils.ts

```diff
@@ -180,8 +219,9 @@ export async function getResults(
     accessType,
     complianceType
   )
-  const queryResult = await queryMetadata(searchQuery, cancelToken)
-  return queryResult
+  const query =
+    faceted === 'true' ? { ...searchQuery, aggs: facetedQuery() } : searchQuery
```

Comment:
Is there a re reason why a boolean is used as string here?

Reply:
The reason is consistency with the existing code and maybe reuse in the future, the params of the function come from the query string of the url, therefore they are a string.

Reply:
Made it a boolean